### PR TITLE
collide: Expose CollisionPolygon's setup_points and verify_points to Python

### PR DESCRIPTION
**Pull request by darktohka**
## Issue description
I would like to set a CollisionPolygon's list of points. Unfortunately, the sequence wrapper over `_points` does not allow this to happen. Fortunately, CollisionPolygon has a `setup_points` method, but this method's visibility is set to private.

## Solution description
This PR sets `setup_points` visibility to public, and adds a Python wrapper for both `setup_points` and `verify_points`, to allow these functions to be called from Python code.

## Checklist
I have done my best to ensure that…
* [X] …I have familiarized myself with the CONTRIBUTING.md file
* [X] …this change follows the coding style and design patterns of the codebase
* [X] …I own the intellectual property rights to this code
* [X] …the intent of this change is clearly explained
* [X] …existing uses of the Panda3D API are not broken
* [x] …the changed code is adequately covered by the test suite, where possible.

### CODE REVIEW

**Review by rdb — CHANGES_REQUESTED** [review]
Looks good, except this won't work in double-precision mode as-is because your code uses LPoint3f whereas the function takes LPoint3.

You may need to disambiguate between Dtool_LPoint3f and Dtool_LPoint3d based on STDFLOAT_DOUBLE since there is no Dtool_LPoint3.

It should not be hard to add unit tests for this.

File: panda/src/collide/collisionPolygon_ext.h [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+ * This class defines the extension methods for CollisionPolygon, which are called
+ * instead of any C++ methods with the same prototype.
+ *
+ * @since 1.11.0
+ */
+template<>
+class Extension<CollisionPolygon> : public ExtensionBase<CollisionPolygon> {
+public:
+    bool verify_points(PyObject *points);
+    void setup_points(PyObject *points);
```
Comment (rdb) [inline panda/src/collide/collisionPolygon_ext.h:35]: Too much indent

**Comment by darktohka** [comment]
The requested changes have been made. Unit tests have been added for both methods, the distinction between Point3d and Point3f is now made, and verify_points is now static.

**Review by rdb — APPROVED** [review]
LGTM

**Pull request closed without merging**